src/Perspective.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-underscore-dangle */
+import Flicking, { EVENTS, Plugin } from "@egjs/flicking";
+
+interface PerspectiveOptions {
+  selector: string;
+  scale: number;
+  rotate: number;
+  perspective: number;
+}
+
+/**
+ * You can apply perspective effect while panel is moving.
+ * @ko 패널들을 움직이면서 입체감을 부여할 수 있습니다.
+ * @memberof Flicking.Plugins
+ */
+class Perspective implements Plugin {
+  private _flicking: Flicking | null;
+
+  /* Options  */
+  private _selector: PerspectiveOptions["selector"];
+  private _scale: PerspectiveOptions["scale"];
+  private _rotate: PerspectiveOptions["rotate"];
+  private _perspective: PerspectiveOptions["perspective"];
+
+  public get selector() { return this._selector; }
+  public get scale() { return this._scale; }
+  public get rotate() { return this._rotate; }
+  public get perspective() { return this._perspective; }
+
+  public set selector(val: string) { this._selector = val; }
+  public set scale(val: number) { this._scale = val; }
+  public set rotate(val: number) { this._rotate = val; }
+  public set perspective(val: number) { this._perspective = val; }
+
+  /**
+   * @param - The selector of the element to which the perspective effect is to be applied. If the selector is blank, it applies to panel element. <ko>입체 효과를 적용할 대상의 선택자. 선택자가 공백이면 패널 엘리먼트에 적용된다.</ko>
+   * @param - Effect amplication scale.<ko>효과 증폭도</ko>
+   * @param - Effect amplication rotate.<ko>회전 증폭도</ko>
+   * @param - The value of perspective CSS property. <ko>css perspective 속성 값</ko>
+   * @example
+   * ```ts
+   * flicking.addPlugins(new Perspective({ selector: "p", scale: 1, rotate: 1, perspective = 1000 }));
+   * ```
+   */
+  public constructor({ selector = "", scale = 1, rotate = 1, perspective = 1000 }: Partial<PerspectiveOptions> = {}) {
+    this._flicking = null;
+    this._selector = selector;
+    this._scale = scale;
+    this._rotate = rotate;
+    this._perspective = perspective;
+  }
+
+  public init(flicking: Flicking): void {
+    if (this._flicking) {
+      this.destroy();
+    }
+
+    this._flicking = flicking;
+
+    flicking.on(EVENTS.MOVE, this._onMove);
+    flicking.on(EVENTS.AFTER_RESIZE, this.update);
```

Comment:
Removing this event handler is missing in `destroy`, can you please add it?

Reply:
Thank you for comment!

In addition to **perspective**, **fade** and **parralex** seemed to have the same problem, so I added them.

Reply:
Great 👍 Thanks for the fix!